samples/zigbee/light_bulb/src/main.c

```diff
@@ -20,8 +20,8 @@
 #include <zboss_api.h>
 #include <zboss_api_addons.h>
 #include <zb_mem_config_med.h>
-#include <zigbee_helpers.h>
-#include <zb_error_handler.h>
+#include <zigbee/zigbee_helpers.h>
+#include <zigbee/zb_error_handler.h>
```

Comment:
I think it would be good to settle on a common prefix for zigbee headers (either `zb_` or `zigbee_`), currently it's a bit messy. There's also an option to drop the prefix, given we need to specify the full path (`zigbe/xxx.h`) anyway.

Reply:
Good idea, sticking with zigbee seems more consistent.